dev/PushNotifications/PushNotifications.idl

```diff
@@ -22,26 +22,29 @@ namespace Microsoft.Windows.PushNotifications
     };
 
     [flags]
-    enum PushNotificationRegistrationOptions
+    enum PushNotificationRegistrationActivators
     {
-        Undefined = 0x0, // Used to check if 0 was passed as a flag
+        Undefined = 0, // Default enum to catch nullptr case when no flag is defined
         PushTrigger = 0x1, // Registers a Push Trigger with Background Infrastructure
-        ComActivator = 0x2, // Registers the Windows App SDK Background Task component as an InProc COM server
-        ProtocolActivator = 0x4 // Registers an application with the PushNotificationsLongRunningTask to be activated via protocol
+        ComActivator = 0x2, // Registers the Background Task component as an in-process COM server
+        ProtocolActivator = 0x4, // Registers an application with the PushNotificationsLongRunningTask to be activated via protocol
     };
 
-    [experimental]
+    // An abstraction over the activation Registration flow
```

Comment:
What does this mean?

Reply:
Not sure why this is there but it should just be experimental.